### ISSUE TITLE
RAHU issues

### ISSUE THREAD

**Issue by pujari** [devksingh4/panchang-calc#1]
File ".\panchang.py", line 593, in navamsa
    nirayana_long = ketu(sidereal_longitude(jd_utc, swe.RAHU))
AttributeError: module 'swisseph' has no attribute 'RAHU'

Any clue how this can be fixed ? Thanks

**Comment by devksingh4** [comment]
Per the documentation, this attribute exists: https://pythonhosted.org/pyswisseph/swisseph-module.html

Have you installed pyswisseph correctly with `pip install pyswisseph`?

**Comment by pujari** [comment]
Yes I did.  pip install pyswisseph and then python .\panchang.py
